<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-GK61F68" ignored="0" name="Default test" tests="1" failures="1" timestamp="2021-01-22T16:43:39 IST" time="37.671" errors="0">
  <testcase name="runScenario" time="33.927" classname="io.cucumber.testng.AbstractTestNGCucumberTests">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertTrue(Assert.java:45)
at org.testng.Assert.assertTrue(Assert.java:55)
at stepDefinition.OraganizationStepDef.i_validate_the_oraganization_created_or_not(OraganizationStepDef.java:47)
at âœ½.I validate the Oraganization created or not(file:///C:/Users/Nitheesha/git/BDDCucumberSDET8/VTIGERBDDframework/./src/test/java/features/Organization.feature:8)
]]>
    </failure>
  </testcase> <!-- runScenario -->
</testsuite> <!-- Default test -->
